### PR TITLE
changes to episode 2, tasks 6-20

### PR DESCRIPTION
**Pull request by mallewellyn**
tasks 6-20 of #114

### CODE REVIEW

File: _episodes_rmd/02-high-dimensional-regression.Rmd [hunk 1]
```diff
@@ -943,7 +945,7 @@ simply due to chance.
 
 > ## Challenge 5
 >
-> 1.  If we run `r nrow(methylation)` tests under the null hypothesis,
+> 1.  If we run `r nrow(methylation)` tests,
```
Comment (alanocallaghan) [inline _episodes_rmd/02-high-dimensional-regression.Rmd:949]:
```suggestion
> 1. If we run `r nrow(methylation)` tests, even if there are no true differences,
```

Comment (alanocallaghan) [reply _episodes_rmd/02-high-dimensional-regression.Rmd:949]: I don't know if this is good phrasing, so feel free to choose your own, but I think we should still keep the spirit of "assuming we're just looking at stochastic variation, and not real signal"

Comment (mallewellyn) [reply _episodes_rmd/02-high-dimensional-regression.Rmd]: Sorry - totally missed this. Have committed below

File: _episodes_rmd/02-high-dimensional-regression.Rmd [hunk 2]
```diff
@@ -221,12 +222,12 @@ to help us understand how ageing manifests.
...
 these. However, high-dimensional data like the ones we're working with
-require some special considerations. A primary consideration, as we saw
+require some special considerations. A first consideration, as we saw
 above, is that there are far too many features to fit each one-by-one as
 we might do when analysing low-dimensional datasets (for example using
 `lm` on each feature and checking the linear model assumptions). A
-secondary consideration is that statistical approaches may behave
-slightly differently in very high-dimensional data, compared to
+second consideration is that statistical approaches may behave
+slightly differently when implemented on very high-dimensional data, compared to
```
Comment (ailithewing) [inline _episodes_rmd/02-high-dimensional-regression.Rmd:231]:
```suggestion
slightly differently when applied to very high-dimensional data, compared to
```
Sounds simpler I think...

**Review by ailithewing — COMMENTED** [review]
Just one comment with v minor change before can be merged.

**Comment by mallewellyn** [comment]
Ah great! See commit above

**Review by ailithewing — APPROVED** [review]

**Pull request merged**